omod/src/test/java/org/openmrs/module/fhir2/providers/PatientFhirResourceProviderTest.java

```diff
@@ -303,14 +316,15 @@ public void searchForPatients_shouldReturnMatchingBundleOfPatientsByCountry() {
 		        .addAnd(new StringOrListParam().add(new StringParam(COUNTRY)));
 		when(patientService.searchForPatients(isNull(), isNull(), isNull(), isNull(), isNull(), isNull(), isNull(), isNull(),
 		    isNull(), isNull(), isNull(), argThat(is(countryParam)), isNull()))
-		            .thenReturn(Collections.singletonList(patient));
+		            .thenReturn(new BaseFhirIBundleResourceProviderTest<>(Collections.singletonList(patient), 10, 1));
 		
-		Bundle results = resourceProvider.searchPatients(null, null, null, null, null, null, null, null, null, null, null,
-		    countryParam, null);
+		IBundleProvider results = resourceProvider.searchPatients(null, null, null, null, null, null, null, null, null, null,
+		    null, countryParam, null);
 		
 		assertThat(results, notNullValue());
-		assertThat(results.isResource(), is(true));
-		assertThat(results.getEntry().size(), greaterThanOrEqualTo(1));
+		assertThat(results.getResources(0, 1), hasSize(equalTo(1)));
+		assertThat(results.getResources(0, 1).get(0).fhirType(), is("Patient"));
+		assertThat(results.getResources(0, 1).get(0).getIdElement().getIdPart(), is(PATIENT_UUID));
```

Comment:
I would probably prefer caching the call of `results.getResources(0, 1)` and reusing that as each time we invoke that, it makes a query to the database and, while it's not a huge deal, I'd like to keep the unit tests as quick as possible so people continue to *want* to run them.